### PR TITLE
Formating the whole Article tables and links.

### PR DESCRIPTION
**Pull request by j0rt3g4**
Created the whole article using markdown instead of HTML.

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Solving issue #3751

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/identity-protection/access-control/special-identities.md [hunk 1]
```diff
@@ -34,364 +34,153 @@ Although the special identity groups can be assigned rights and permissions to r
 
 For information about security groups and group scope, see [Active Directory Security Groups](active-directory-security-groups.md).
 
-The special identity groups are described in the following tables.
+The special identity groups are described in the following tables.
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/special-identities.md]:
```suggestion
The special identity groups are described in the following tables:
```

**Comment by mypil** [comment]
@Justinha - Can you please assist to check, approve and merge the content of this PR based on issue #3751?

Thank you.

**Comment by j0rt3g4** [comment]
@Justinha The changes are done. Thank you!

**Pull request merged**